src/components/AvatarGroup/AvatarGroupCounterTooltipHelper.js

```diff
@@ -4,88 +4,103 @@ import Flex from "../Flex/Flex";
 import Avatar from "../Avatar/Avatar";
 import ClickableWrapper from "../Clickable/ClickableWrapper";
 import avatarGroupCounterTooltipContentStyles from "./AvatarGroupCounterTooltipContent.module.scss";
+import useEventListener from "../../hooks/useEventListener";
+import { useListenFocusTriggers } from "../../hooks/useListenFocusTriggers";
 
 const KEYS = ["Tab"];
+const ESC = ["Escape"];
 
 export function useTooltipContentTabNavigation({
   counterContainerRef = undefined,
   tooltipContentContainerRef,
   focusPrevPlaceholderRef,
   focusNextPlaceholderRef,
-  setShouldUpdate,
-  setIsTooltipVisible
+  isKeyboardTooltipVisible,
+  setIsKeyboardTooltipVisible
 }) {
-  const hideTooltip = useCallback(() => {
-    // Tricky way to close the tooltip
-    setTimeout(() => {
-      setIsTooltipVisible(false);
-      setIsTooltipVisible(true);
-    });
-  }, [setIsTooltipVisible]);
+  const showKeyboardTooltip = useCallback(() => {
+    if (!isKeyboardTooltipVisible) setIsKeyboardTooltipVisible(true);
```

Comment:
There is an eye-catchable difference between tooltip opening for Avatar and AvatarGroupCounter - for AvatarGroupCounter it's opening without delay. Let's try to make it consistent